packages/pocket-ic/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for verified application subnets: the library function `PocketIcBuilder::with_verified_application_subnet` adds a verified application subnet to the PocketIC instance;
   the library function `PocketIc::get_verified_app_subnets` lists all verified application subnets of the PocketIC instance.
 - The function `PocketIcBuilder::with_log_level` to specify the replica log level of the PocketIC instance.
+- The function `PocketIcBuilder::with_config` to specify a custom `ExtendedSubnetConfigSet`.
+  This function `PocketIcBuilder::with_config` can only be used if no config was (partially) provided so far.
```

Comment:
If this introduces an illegal way to compose `.with_x`s, we should maybe avoid that and go for something like 
`fn PocketIcBuilder::new(cfg: Option<ExtSubCfgSet>)`, so the Builder is either constructed with it or not, and then all other .with_x fully compose. Or am I missing some other incompatible things?

Reply:
sounds dangerously like a builder-builder

Reply:
> fn PocketIcBuilder::new(cfg: Option<ExtSubCfgSet>)

Great suggestion! Let me add a new construction `new_with_config` in addition to the existing one.

Reply:
Sorry I jumped to conclusions before reading code. 

The builder already takes an Option<Config...>, you are merely offering "with_config" can only be used if that initial argument was None, correct?

Reply:
Otherwise, the config would overwrite any changes you did (which might be confusing/misleading) or we'd need to merge the two configs (which is unnecessary complexity in my opinion).